src/appshell/view/appmenumodel.h

```diff
@@ -73,6 +75,7 @@ class AppMenuModel : public muse::uicomponents::AbstractMenuModel
 
     muse::uicomponents::MenuItem* makeFileMenu();
     muse::uicomponents::MenuItem* makeEditMenu();
+    void updateUndoRedoItems();
```

Comment:
For some reason, this line here bugs me because it sticks out so much. I get why it's there though, and though it matters not at all, I would probably move the declaration to the bottom to avoid disrupting the march of top-level menu-maker functions. This is totally your call.

Reply:
Yeah, I didn't know very well what to do with it, but perhaps the bottom is the best place indeed.